DUI3-DX/Connectors/Autocad/Speckle.Connectors.AutocadShared/DependencyInjection/AutofacAutocadModule.cs

```diff
@@ -50,7 +50,7 @@ protected override void Load(ContainerBuilder builder)
     // Operations
     builder.RegisterType<SendOperation<(DBObject obj, string applicationId)>>().InstancePerLifetimeScope();
     builder.RegisterType<ReceiveOperation>().SingleInstance();
-    builder.RegisterType<SyncToUIThread>().As<ISyncToMainThread>().SingleInstance();
+    builder.RegisterType<SyncToUIThread>().As<ISyncToMainThread>().SingleInstance().AutoActivate(); // Auto activation makes SynchronizationContext available right after building. Otherwise we were missing it in UoWs.
```

Comment:
Does this need propagating to Rhino/Revit?

Reply:
I believe it is more of a MainThread thing, Rhino should be fine without AutoActive, but not sure about Revit. And @connorivy was saying he did something Revit specific? If `ISyncToMainThread` interface work with Revit too, we should align it I believe.

Reply:
Ok no blocker for this then! In we go!